pkg/purser_plugin/commons.go

```diff
@@ -36,3 +41,27 @@ func executeCommand(command string) []byte {
 	}
 	return out.Bytes()
 }
+
+func currentMonthActiveTimeInHours(startTime, endTime meta_v1.Time) float64 {
+	now := time.Now()
+	monthStart := meta_v1.Date(now.Year(), now.Month(), 1, 0, 0, 0, 0, time.Local)
```

Comment:
Create a method, so that it can be reusable.

Reply:
`now := time.Now()` has been used in multiple places repeatedly. In order to avoid code duplicity we should abstract it and store it as a global variable or have a dedicated function returning it.